_episodes/06-raster-intro.md

```diff
@@ -0,0 +1,433 @@
+
+We will be exploring this data throughout this episode. By the end of this episode, you will be able to understand and explain the metadata output.
+
+> ## Tip - Variable names
+> To improve code readability, file and object names should be used that make it clear what is in the file. The data for this episode covers Amsterdam, and is from Band 9, so we'll use a naming convention of `raster_ams_b9`.
+{: .callout}
+
+## Visualize a Raster
+
+After viewing the attributes of our raster, we can examine the raw values of the array with `.values`:
+
+~~~
+raster_ams_b9.values
+~~~
+{: .language-python}
+~~~
+array([[[    0,     0,     0, ...,  8888,  9075,  8139],
+        [    0,     0,     0, ..., 10444, 10358,  8669],
+        [    0,     0,     0, ..., 10346, 10659,  9168],
+        ...,
+        [    0,     0,     0, ...,  4295,  4289,  4320],
+        [    0,     0,     0, ...,  4291,  4269,  4179],
+        [    0,     0,     0, ...,  3944,  3503,  3862]]], dtype=uint16)
+~~~
+{: .output}
+
+This can give us a quick view of the values of our array, but only at the corners. Since our raster is loaded in python as a `DataArray` type, we can plot this in one line similar to a pandas `DataFrame` with `DataArray.plot()`.
+
+~~~
+raster_ams_b9.plot()
+~~~
+{: .language-python}
+
+<img src="../fig/E06-01-overview-plot-B09.png" title="Raster plot with rioxarray" alt="Raster plot with defualt setting" width="612" style="display: block; margin: auto;" />
+
+Nice plot! Notice that `rioxarray` helpfully allows us to plot this raster with spatial coordinates on the x and y axis (this is not the default in many cases with other functions or libraries).
+
+This plot shows the satellite measurement of the spectral band `B09` for an area that covers part of the Netherlands. According to the [Sentinel-2 documentaion](https://sentinels.copernicus.eu/web/sentinel/technical-guides/sentinel-2-msi/msi-instrument), this is a band with the central wavelength of 945nm, which is sensitive to water vapor. It has a spatial resolution of 60m. Note that the `band=1` in the image title indicates the total number of bands in the `DataArray`, but not the Sentinel-2 band number `B09`. 
+
+In a quick view of the image, we can notice that half of the image is blank. We also see that the pixels with high reflectance values are the clouds at the top, and the contrast of everything else is quite low. This is expected because this band is sensitive to the water vapor. However if one would like to have a better color contrast, one can add the option `robust=True`, which displays values between the 2nd and 98th percentile:
```

Comment:
```suggestion
With a quick view of the image, we notice that half of the image is blank, no data is captured. We also see that the cloudy pixels at the top have high reflectance values, while the contrast of everything else is quite low. This is expected because this band is sensitive to the water vapor. However if one would like to have a better color contrast, one can add the option `robust=True`, which displays values between the 2nd and 98th percentile:
```

Reply:
do we want to mention in this lesson how to clip by other percentile values? this was mentioned in the old version by necessity since robust = True didn't always lead to a good viz

Reply:
maybe in a tooltip?

Reply:
Hi Ryan, thx for the comment! Indeed `robust=True` does not always work. The other away I can think of is specifying the `vmin` and `vmax` value directly. Then one need to convert the percentile to these two values. I will briefly add one line mentioning this.